scripts/components/BucketCollections.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import { Link } from "react-router";
+
+import BucketTabs from "./BucketTabs";
+
+
+class DataList extends Component {
+  render() {
+    const {
+      bid,
+      data,
+      listLoaded,
+    } = this.props;
+    return (
+      <table className="table table-striped table-bordered record-list">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Schema</th>
+            <th>Cache Expires</th>
+            <th>Last mod.</th>
+          </tr>
+        </thead>
+        <tbody className={!listLoaded ? "loading" : ""}>{
+          data.map((collection, index) => {
```

Comment:
You can destructure the arg here:

```jsx
data.map(({id, schema, cache_expires, last_modified}), index) => {
```

Alternatively (probably prefered), destructuring the const:

```jsx
data.map((collection, index) => {
  const {id, schema, cache_expires, last_modified} = collection;
```